spec/services/request_create_service_spec.rb

```diff
@@ -51,7 +51,6 @@
             :state             => Request::STARTED_STATE,
             :decision          => Request::UNDECIDED_STATUS,
             :reason            => nil,
-            :random_access_key => be_kind_of(String),
```

Comment:
not from this PR

Reply:
Changed for spec test failure.

Reply:
you should not test with code merged from pending #297